### ISSUE TITLE
Calendar - update colors to be accessible

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#7467]
Based on the work done in https://github.com/grommet/grommet/issues/7423 the Calendar colors should be updated to meet accessibility requirements.

Color recommendations here: https://www.figma.com/design/OBxHRrYylrg1IyNetpK7OF/%237423---Calendar-contrast?node-id=45-31

**Comment by jcfilben** [comment]
Re-opened #7423, need that ticket closed before continuing
